projectmanagement/src/main/java/life/qbic/projectmanagement/application/sample/SamplePreview.java

```diff
@@ -146,6 +154,12 @@ public String specimen() {
   public String analyte() {
     return analyte;
   }
+  public String analysisType() {
+    return analysisType;
+  }
+  public String comment() {
+    return comment;
+  }
```

Comment:
As they are optional, why not return an Optional here?
```suggestion
  public Optional<String> analysisType() {
    return Optional.ofNullable(analysisType);
  }
  public Optional<String> comment() {
    return Optional.ofNullable(comment);
  }
```